packages/chat-component/src/main.ts

```diff
@@ -119,6 +133,7 @@ export class ChatComponent extends LitElement {
         // non-streamed response
         const processedText = processText(message, [citations, followingSteps, followupQuestions]);
         message = processedText.replacedText;
+        console.log(message, '####MESSAGE####');
```

Comment:
should be removed I guess?

Reply:
Ooops. We should catch them with the pre-commit hook. We are almost shipping one with every PR. :)

Reply:
We should add the https://eslint.org/docs/latest/rules/no-console eslint hook then!